### ISSUE TITLE
WARNING: The html argument of XMLParser() is deprecated

### ISSUE THREAD

**Issue by egberts** [brunato/xmlschema#97]
```
/Users/jdoe/work/python/env3/lib/python3.7/site-packages/defusedxml-0.5.0-py3.7.egg/defusedxml/ElementTree.py:68
  /Users/jdoe/work/python/env3/lib/python3.7/site-packages/defusedxml-0.5.0-py3.7.egg/defusedxml/ElementTree.py:68: DeprecationWarning: The html argument of XMLParser() is deprecated
    _XMLParser.__init__(self, html, target, encoding)
```

**Comment by brunato** [comment]
Hi,
this warning, caused by *defusedxml* library, would be resolved with the next release of Python 3.7 (3.7.2 that is soon to be released), that embed the necessary fixes to process XML data.
See https://github.com/tiran/defusedxml/issues/27. 

So the *defuse* option and the usage of the *defusedxml* library is going to be removed soon, progressively.

**Comment by brunato** [comment]
The fixes included in last versions of Python 3.6 and 3.7 don't cover XML entities vulnerabilities (see https://bugs.python.org/issue17239 and https://bugs.python.org/issue34623).

So the *defusedxml* library, that seems to be no longer maintained, will be replaced in the next release by a custom `SafeXMLParser` for ElementTree, taylored for the needs of *xmlschema*.

**Comment by brunato** [comment]
The new release v1.0.8 contains a custom XMLParser for protecting from XML entities injection. The new ``SafeXMLParser`` doesn't use the *html* argument, so the deprecation warning should not appear again.
